package.json

```diff
@@ -33,7 +33,7 @@
     "cleanup": "./cleanup.sh",
     "build": "yarn workspaces run build",
     "build:apidocs": "rm -rf docs; cd packages/algob; typedoc --out docs --excludePrivate --exclude 'src/internal/**' --mode modules --gitRevision typedoc; touch docs/.nojekyll && mv docs ../../",
-    "watch": "tsc --build --watch packages/algob",
+    "watch": "tsc --build --watch packages/runtime packages/algob",
```

Comment:
how about adding types-algosdk as well ?

Reply:
adding.

Reply:
Update: Checked while updating `types-algosdk` (without `watch`), and the types are updated in `/packages` in realtime.

Reply:
ok